libmscore/articulation.cpp

```diff
@@ -135,6 +135,16 @@ bool Articulation::readProperties(XmlReader& e)
         if (id == SymId::noSym) {
             id = oldArticulationNames2SymId(s);             // compatibility hack for "old" 3.0 scores
         }
+        if (id == SymId::noSym || s == "ornamentMordentInverted") {   // SMuFL < 1.30
+            id = SymId::ornamentMordent;
+        }
+
+        QString programVersion = masterScore()->mscoreVersion();
+        if (!programVersion.isEmpty() && programVersion < "3.6") {
```

Comment:
This might need to get adjusted, depending on when and were this PR gets merged

Reply:
As the corresponding PR had been merged into 3,x, this here is correct and can stay as is.